src/App.css

```diff
@@ -2,6 +2,73 @@
     text-align: center;
 }
 
+/*
+Desktop Header
+ */
+.categories {
+    display: inline-block;
+    text-transform: uppercase;
+    color: black;
+    cursor: pointer;
+    height: 37px !important;
+}
+
+.categories > h4 {
+    margin: 10px 5px;
+    font-size: 12px;
+}
+
+.nav {
+    display: inline-block;
+    float: right;
+}
+
+.appBar {
+    background-color: black !important;
+    display: table !important;
+}
+
+#explore {
+    background-color: #8acc25;
+}
+
+#report {
+    background-color: #ffd700;
+}
+
+#resources {
+    background-color: #39c9bb;
+}
+
+.headerTitle {
+    display: inline;
+    font-size: 14px;
+    color: white;
+    position: absolute;
+    left: 27%;
+}
+
+@media (min-width: 960px) {
+    .headerTitle {
+        left: 24%;
+    }
+}
+
+@media (min-width: 1440px) {
```

Comment:
This is really slick. Are 960/1440px pretty standard values for these kinds of breakpoints? Also 20/24/27% or did you come up with that on your own?

Reply:
Mmm, I guess we should actually standardize these some more.

Typically we use 768px and 1280px etc. (https://medium.com/@uiuxlab/the-most-used-responsive-breakpoints-in-2017-of-mine-9588e9bd3a8a)

I'm going to rework these a bit. 

The percentages were defined by me to accommodate the breakpoints.